packages/sources/sportsdataio/README.md

```diff
@@ -129,9 +130,45 @@ Get the MLB schedule on a given date
 
 #### Input Parameters
 
-| Required? | Name |                Description                 | Options | Defaults to |
-| :-------: | :--: | :----------------------------------------: | :-----: | :---------: |
-|    ✅     | date | The date games to query were/are played on |         |             |
+| Required? |      Name      |                                      Description                                      |          Options           | Defaults to |
+| :-------: | :------------: | :-----------------------------------------------------------------------------------: | :------------------------: | :---------: |
+|    ✅     |      date      |                      The date games to query were/are played on                       |                            |             |
+|           |     teamID     |                           The team ID to filter by e.g LAD                            | See list of team IDs below |             |
+|           | onlyShowGameID | Set to true if response should only contain game IDs instead of the whole game object |                            |             |
+
+#### List of team IDs
```

Comment:
Who will maintain this list? Can we link to any docs from SDIO instead of taking this responsibility?